backend/de.metas.contracts/src/main/java/de/metas/contracts/FlatrateTermPricing.java

```diff
@@ -84,20 +85,31 @@ public IPricingResult computeOrThrowEx()
 
 	private PriceListId retrievePriceListForTerm()
```

Comment:
it's ok since you jsut bug-fixed this class, but when changing new stuff or when we have the time, we avoid using services in classes like this one. teo can tell you more (spoiler: it's DDD)

Reply:
I know 😎